torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8138,19 +8138,19 @@ def sample_inputs_lerp(op_info, device, dtype, requires_grad, **kwargs):
         # no broadcast
         SampleInput(make_arg((S, S)), args=(make_arg((S, S)), 0.4)),
         # broadcast rhs
-        SampleInput(make_arg((S, S)), args=(make_arg((S,)), 0.4)),
+        SampleInput(make_arg((S, S)), args=(make_arg((S,)), 0.6)),
         # scalar tensor
         SampleInput(make_arg(()), args=(make_arg(()), 0.4)),
         # broadcast rhs scalar-tensor
-        SampleInput(make_arg((S, S)), args=(make_arg(()), 0.4)),
+        SampleInput(make_arg((S, S)), args=(make_arg(()), 0.6)),
```

Comment:
Check case where `weight` is greater than 0.5. (There are two paths based on weight)